pwndbg/constants/ptmalloc.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import platform
+
+import gdb
+
+import pwndbg.arch
+
+# Heap flags
+PREV_INUSE      = 1
+IS_MMAPPED      = 2
+NON_MAIN_ARENA  = 4
+SIZE_BITS       = ( PREV_INUSE | IS_MMAPPED | NON_MAIN_ARENA )
+
+# Constants and macros taken from glibc malloc.c
+MIN_CHUNK_SIZE      = pwndbg.arch.ptrsize * 4
```

Comment:
`pwndbg.arch.ptrsize` is not a global constant, this will change based on the current binary (e.g. `set arch`).  This unfortunately has to be a function, or needs an additional `event` to update the global.  See how `pwndbg.arch` handles this.